parent-pom/pom.xml

```diff
@@ -201,6 +201,13 @@
                 <artifactId>maven-plugin-integration-testing</artifactId>
                 <version>1.1.2</version>
                 <scope>test</scope>
+                <exclusions>
```

Comment:
Why has this not been necessary before?

Reply:
It was not necessary. But now I was annoyed that my IDE suggested to import the wrong `@Test` annotation ;)